app/src/models/avatar.ts

```diff
@@ -84,10 +85,18 @@ export function getAvatarUsersForCommit(
     )
   )
 
+  coAuthoredByCommitter = commit.coAuthors.some(
```

Comment:
Seeing how this variable is only computed once and then never assigned to again I think we should make it a constant and drop the `let` above.

```suggestion
  const coAuthoredByCommitter = commit.coAuthors.some(
```

Making it a `const` signals to people (and computers 😛) reading the code that it's immutable and therefore not going to change at any point after it's been assigned to vs `let` where they might have to read the entire method to see if it's being changed anywhere else.

Reply:
Thanks again, I've updated